src/wiktextract/extractor/en/example.py

```diff
@@ -54,6 +55,25 @@ def extract_example_list_item(
                     )
             else:
                 examples.append(q_example)
+        elif template_node.template_name in [
+            "ux",
+            "usex",
+            "uxi",
+            "ko-usex",
+            "koex",
+            "ko-x",
+            "th-usex",
+            "th-x",
+            "th-xi",
```

Comment:
These templates will be handled in this file in order to add the bold word offsets data.